packages/components/src/index.ts

```diff
@@ -1,9 +1,25 @@
 // TODO: Remove when Date Picker is migrated to KAIO
 export * from "./types/DatePicker"
 
+export * from "./Avatar"
+export * from "./AvatarGroup"
+export * from "./Badge"
+export * from "./BrandMoment"
 export * from "./Button"
+export * from "./ButtonGroup"
+export * from "./Card"
+export * from "./Checkbox/Checkbox"
+export * from "./Checkbox/CheckboxField"
+export * from "./Checkbox/CheckboxGroup"
```

Comment:
Oh? We don't want `Checkbox` to be an entrypoint for all 3 so that here can just draw from that? I think that's what I did for `FilterButton`

Reply:
Debated wether or not to change it now, I guess it's not too difficult :P